File: 0268-missing-number/0268-missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        numbers_sorted = sorted(nums)
        n = len(nums)
        i = 0
        while i <= n:
            if i == n or i != numbers_sorted[i]:
                return i
            i = i + 1
        

File: 0268-missing-number/0268-missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    let sorted = nums.sort((a, b) => a - b);
    
    for (let i = 0; i <= sorted.length; i++) {
        if (i !== sorted[i]) {
            return i;
        }
    }
};